lib/ui/views/home/home_view.dart

```diff
@@ -9,30 +9,48 @@ import 'package:veggie_go_malaysia/ui/views/home/home_viewmodel.dart';
 import 'package:veggie_go_malaysia/ui/views/home/widgets/location_bar.dart';
 import 'package:veggie_go_malaysia/ui/views/home/widgets/quick_search.dart';
 
+import '../../../constants/colors.dart';
+import 'widgets/quick_search.dart';
 import 'widgets/restaurant_card/restaurant_card.dart';
 
 class HomeView extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     return ViewModelBuilder<HomeViewModel>.reactive(
       builder: (context, model, child) => Scaffold(
+        backgroundColor: ThemeColors.background,
         body: SafeArea(
-          child: Padding(
-            padding: EdgeInsets.symmetric(horizontal: 80.w),
-            child: SingleChildScrollView(
-              child: Column(
-                children: <Widget>[
-                  Padding(
-                    padding: EdgeInsets.symmetric(vertical: 80.h),
-                    child: _SearchBar(),
+          child: CustomScrollView(
+            slivers: <Widget>[
+              SliverAppBar(
```

Comment:
@joekendal  I think we cannot use SliverAppBar here as the Scaffold will be changed to CupertinoTabScaffold later during BottomNavbar implementation.

CupertinoSliverAppBar should be used instead. 

Basically, replace every materials widgets to Cupertino widgets.

Reply:
@alvindrakes they aren't simply interchangeable... CupertinoSliverAppBar does not exist and for a reason. Flutter does not use native widgets, it simply paints something that looks like them. Maybe you meant using a CupertinoSliverNavigationBar? if so then we need to rethink the design... This does not seem to enable the kind of behaviour wanted. What is the reason for it not being okay to use SliverAppBar exactly? If we want to use only cupertino widgets, then the design for the app needs to be redone with this consideration and restrictions on what can and can't be implemented
![Screenshot 2020-06-05 at 21 18 28](https://user-images.githubusercontent.com/13680617/83919349-52791a00-a772-11ea-90fd-55e98741f2b7.png)
![Screenshot 2020-06-05 at 21 20 39](https://user-images.githubusercontent.com/13680617/83919410-6de42500-a772-11ea-97cb-431fa1637e50.png)

Reply:
Ok. After some discussion. We have decided to use MaterialAPP as the parent widget. 

1 small feedback on ui, instead of showing an arrow btn when searching. I think it will look nicer if you could animate and expand the search bar when clicked to this: 
![Image from iOS](https://user-images.githubusercontent.com/13532268/83939935-57da7100-a813-11ea-8e05-6f5af7c50a89.png)

It looks better visually compared to having a grey box area sliding in.